public/apps/account/tenant-switch-panel.tsx

```diff
@@ -212,11 +212,8 @@ export function TenantSwitchPanel(props: TenantSwitchPanelProps) {
         await changeTenant(tenantName);
         props.handleClose();
       } catch (e) {
-        if (e.body) {
-          setErrorCallOut('Failed to switch tenant due to ' + e.body?.message);
-        } else {
-          setErrorCallOut('Failed to switch tenant. ' + GENERIC_ERROR_INSTRUCTION);
-        }
+        console.error(e);
+        setErrorCallOut(constructErrorMessage(e, 'Failed to switch tenant.'));
```

Comment:
For awareness,  if e.body is given, the error message is slightly changed ("due to" is gone).

Reply:
I removed it intentionally. This is because in case of empty message, it will show the generic instruction. Thus the "due to" sounds weird. After removing it, the full message on the UI still reads like a well structured paragraph.